### PR TITLE
Point students to the xrust Repository

### PR DESCRIPTION
**Pull request by IanWhitney**
I was trying to figure out how how I even found this repository. I think
I stumbled into it via the contributing guide on the main exercism repo.

I thought it would be nice to have each exercise contain a pointer to
the repo, so that students can easily see where all this code comes
from.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -20,5 +20,11 @@ if you wish.
 Make sure to read the [Crates and Modules](https://doc.rust-lang.org/stable/book/crates-and-modules.html) chapter if you
 haven't already, it will help you with organizing your files.
 
+## Feedback, Issues, Pull Requests
+
+The [exercism/xrust](https://github.com/exercism/xrust) repository on GitHub is the home for all of the Rust exercises. If you have feedback about an exercise, or want to help implement new exercises, head over there and create an issue. Members of the [rust track team](https://github.com/orgs/exercism/teams/rust) are happy to help!
```
Comment (petertseng) [inline SETUP.md]: rust team *might* only visible to those who are already members of the org. will check with a logged out win shortly

Comment (petertseng) [inline SETUP.md]: yeah asks me to log in. Not sure what's the best way to deal with this

**Comment by petertseng** [comment]
hmm, seems like a thing that might be good to have in all the repos. wonder if we can make the site automatically generate this for all

File: SETUP.md (hunk 1 above)
Comment (IanWhitney) [inline SETUP.md]: Dang. I'll remove that part, I guess.

**Comment by IanWhitney** [comment]
Hey @kytrinyx, @petertseng has a good idea. Can we do this automatically? 

Also, any ideas on how we publicly share the track maintainer team link?

**Comment by kytrinyx** [comment]
I've added an issue to create an overview of important repositories here: https://github.com/exercism/exercism.io/issues/2924

The idea is that we can link to it from the new contributing page that's being worked on here:
https://github.com/exercism/exercism.io/pull/2918

Also, we should probably stick something at the bottom of the main code page that links to the relevant repository, as well as make sure that if you go to the exercism.io/languages/TRACK_ID page, that it is easy to find the repository.

I'm not sure about the gitter thing—I think we should definitely figure that out and standardize it (and script it) in some way. Want to open something in https://github.com/exercism/discussions/issues so that we can get everyone who cares about this involved over there?

I'll ask some folks at GitHub about a public link to a team. That would be helpful.

**Pull request merged**